frontend/packages/lm-eval/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Source code for the repos
+ARG UI_SOURCE_CODE=./upstream/frontend
+ARG BFF_SOURCE_CODE=./upstream/bff
+
+# Set the base images for the build stages
+ARG NODE_BASE_IMAGE=node:20
```

Comment:
There are several tweaks that we are performing to the dockerfile, you can take a look at them here: https://github.com/opendatahub-io/odh-dashboard/pull/4584